datafusion/core/tests/sql/expr.rs

```diff
@@ -247,8 +247,8 @@ async fn query_not() -> Result<()> {
 async fn csv_query_sum_cast() {
     let ctx = SessionContext::new();
     register_aggregate_csv_by_sql(&ctx).await;
-    // c8 = i32; c9 = i64
-    let sql = "SELECT c8 + c9 FROM aggregate_test_100";
+    // c8 = i32; c6 = i64
```

Comment:
I made this change due to the fact that https://github.com/apache/arrow-datafusion/pull/3359 changed the type of `c9` so it was no longer i64 but u64